index.html

```diff
@@ -8,6 +8,8 @@
     <link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
     <link rel="stylesheet" href="./main.css">
     <meta name="theme-color" content="#ffffff">
+    <meta name="apple-mobile-web-app-capable" content="yes">
```

Comment:
Already done in #45 😄 

If you pinned it before it’s merged, you need to unpin it and pin it again.